src/Compilers/CSharp/CscCore/CscCore.csproj

```diff
@@ -15,7 +15,7 @@
     <LargeAddressAware>true</LargeAddressAware>
     <StartupObject>Microsoft.CodeAnalysis.CSharp.CommandLine.Program</StartupObject>
     <SolutionDir Condition="'$(SolutionDir)' == '' OR '$(SolutionDir)' == '*Undefined*'">..\..\..\..\</SolutionDir>
-    <OutDir>$(OutDir)core-clr\</OutDir>
+    <OutDir>$(OutDir)csccore\</OutDir>
```

Comment:
Can we have a root directory name coreclr and have the projects build individuually under them?